tracer/src/Datadog.Trace/Iast/HashBasedDeduplication.cs

```diff
@@ -3,38 +3,51 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+using System;
 using System.Collections.Generic;
 
 namespace Datadog.Trace.Iast;
```

Comment:
#nullable enable missing here
```suggestion
#nullable enable 
namespace Datadog.Trace.Iast;
```